ui/components/ui/box/README.mdx

```diff
@@ -0,0 +1,124 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import ActionableMessage from '../actionable-message';
+
+import Box from '.';
+
+# Box
+
+Box is a utility component that can be used for layout or as a base for other UI components.
+
+<Canvas>
+  <Story id="ui-components-ui-box-box-stories-js--default-story" />
+</Canvas>
+
+## Component API
+
+| Name            | Description                         | Default            |
+| --------------- | ----------------------------------- | ------------------ |
+| children        | node func                           | -                  |
+| flexDirection   | Object.values(FLEX_DIRECTION)       | FLEX_DIRECTION.ROW |
+| flexWrap        | Object.values(FLEX_WRAP)            | -                  |
+| gap             | 1,2,4,6,8                           | -                  |
+| margin          | 1,2,4,6,8 or array of numbers [1,2] | -                  |
+| marginTop       | 1,2,4,6,8                           | -                  |
+| marginBottom    | 1,2,4,6,8                           | -                  |
+| marginRight     | 1,2,4,6,8                           | -                  |
+| marginLeft      | 1,2,4,6,8                           | -                  |
+| padding         | 1,2,4,6,8 or array of numbers [1,2] | -                  |
+| paddingTop      | 1,2,4,6,8                           | -                  |
+| paddingBottom   | 1,2,4,6,8                           | -                  |
+| paddingRight    | 1,2,4,6,8                           | -                  |
+| paddingLeft     | 1,2,4,6,8                           | -                  |
+| borderColor     | Object.values(COLORS)               | -                  |
+| borderWidth     | number                              | -                  |
+| borderRadius    | Object.values(SIZES)                | -                  |
+| borderStyle     | Object.values(BORDER_STYLE)         | -                  |
+| alignItems      | Object.values(ALIGN_ITEMS)          | -                  |
+| justifyContent  | Object.values(JUSTIFY_CONTENT)      | -                  |
+| textAlign       | Object.values(TEXT_ALIGN)           | -                  |
+| display         | Object.values(DISPLAY)              | -                  |
+| width           | Object.values(BLOCK_SIZES)          | -                  |
+| height          | Object.values(BLOCK_SIZES)          | -                  |
+| backgroundColor | Object.values(COLORS)               | -                  |
+| className       | string                              |                    |
```

Comment:
Had to add the args table manually here as the storybook `ArgsTable` component doesn't like line `67 - 79` in `box.js` for some reason

<img width="1440" alt="Screen Shot 2022-02-23 at 8 52 49 AM" src="https://user-images.githubusercontent.com/8112138/155250528-ba0fce97-dfd7-4344-a98b-c25eb2971715.png">